test/app-tests/loading-test.js

```diff
@@ -94,13 +81,14 @@ describe('loading:', function() {
         expect(MatrixClientPeg.get()).toBe(null);
 
         // chrome seems to take *ages* to delete the indexeddbs.
-        this.timeout(10000);
+        await sleep(10000);
```

Comment:
Tests aren't running in chrome anymore, is this still needed?

Reply:
nope, apparently not. Have removed.